examples/siro_sandbox/controllers/controller_abc.py

```diff
@@ -215,12 +214,10 @@ def act(self, obs, env):
         else:
             step_data = [a.item() for a in action_data.env_actions.cpu()]
 
-        action = gym_wrapper.continuous_vector_action_to_hab_dict(
-            self._env_spec.orig_action_space,
-            self._env_spec.action_space,
-            step_data[0],
-        )
+        action = step_data[0]
```

Comment:
How about assert `len(action_data.env_actions) == 1` and then directly grab the first item? If we know there's only one environment, let's assert that early. Let's not construct unnecessary one-element lists.